app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -105,6 +118,13 @@ class PocketVideoFetchScheduler(
         // Since this is a background job, we're in no rush and can wait a while.
         @VisibleForTesting(otherwise = PRIVATE) val BACKOFF_DELAY_MIN_MILLIS = TimeUnit.SECONDS.toMillis(30)
         @VisibleForTesting(otherwise = PRIVATE) val BACKOFF_DELAY_MAX_MILLIS = TimeUnit.SECONDS.toMillis(60)
+
+        private var DELAY_TIME_FOR_QA_MILLIS = TimeUnit.SECONDS.toMillis(5)
```

Comment:
Since this is a QA feature, maybe we're better off initializing this to null, or -1, and asserting this has been set before we these values. It'll help ensure that, while the code changes, our assumptions about how these values are being used won't change. This assumes we keep `isQA` or have another way of representing that state outside this value.